### PR TITLE
Differentiate between computed and uncomputed member expressions

### PR DESCRIPTION
**Pull request by bomsy**
Associated Issue: #4020

### Summary of Changes

* Add the computed property to the member expression objects returned from getSymbols
* Differentiate computed from un-computed memberexpressions

Example test plan:

- [x] Debug todomvc example
- [x] open `backbone.js` and set a breakpoint on line 223
- [x] Refresh the debuggee (which hits the breakpoint)
- [x] Hover over `id` in `listeningTo[id] = obj`
- [x] It should show `l2`

### CODE REVIEW

File: src/actions/ast.js [hunk 1]
```diff
@@ -117,7 +117,8 @@ function findBestMatch(symbols, tokenPos, token) {
     const overlaps =
       expression.location.start.line == line &&
       expression.location.start.column <= column &&
-      expression.location.end.column >= column;
+      expression.location.end.column >= column &&
+      !expression.computed;
```
Comment (jasonLaster) [inline src/actions/ast.js]: this is not related to this patch, but it would be nice to move `findBestMatch` to a utility and add some unit tests.

**Comment by jasonLaster** [comment]
Looks great!

File: src/actions/ast.js (hunk 1 above)
Comment (bomsy) [reply src/actions/ast.js]: Yeah! thanks for the confirm, thought about that ..but was wondering if we do it in another PR. i think it'll be cool to do here..

File: src/utils/ast.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      expression.location.end.column >= column &&
+      !expression.computed;
+
+    if (overlaps) {
+      return expression;
+    }
+
+    return found;
+  }, {});
+}
```
Comment (bomsy) [inline src/utils/ast.js]: @jasonLaster was not sure if this is the best place to put this. ur tots!

Comment (jasonLaster) [reply src/utils/ast.js]: not a bad spot :)

**Pull request merged**